lib/matplotlib/type1font.py

```diff
@@ -146,47 +145,52 @@ def _tokens(cls, text):
         A PostScript tokenizer. Yield (token, value) pairs such as
         (_TokenType.whitespace, '   ') or (_TokenType.name, '/Foobar').
         """
+        # Preload enum members for speed.
+        whitespace = _TokenType.whitespace
+        name = _TokenType.name
+        string = _TokenType.string
+        delimiter = _TokenType.delimiter
+        number = _TokenType.number
```

Comment:
These names are pretty generic, and I feel like they could accidentally be re-used by the end of the function, so should get a prefix/suffix of some sort.

Reply:
yup, and below too.

Reply:
Hmm, but you didn't do the below one?

Reply:
oops, fixed.